pkg/cmd/build/view.go

```diff
@@ -97,6 +92,12 @@ func NewCmdBuildView(f *factory.Factory) *cobra.Command {
 				return err
 			}
 
+			if web {
+				buildUrl := fmt.Sprintf("https://buildkite.com/%s/%s/builds/%d", bld.Organization, bld.Pipeline, bld.BuildNumber)
+				fmt.Printf("Opening %s in your browser\n", buildUrl)
+				return browser.OpenURL(buildUrl)
+			}
+
```

Comment:
We also don't need to put this below the spinner do we? Once we resolve the build info, we have enough to construct the URL so can open it straight away and return - so its faster for the user.

Reply:
The reason for it was to ensure that we only open the page in web if there are no API errors. But I am fine also in putting it back up.

Reply:
Oh yeah but the build is resolved so we should try to open it i think

Reply:
I have moved that back up

Reply:
I think you might have not pushed the changes? Its still below